<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Test10_1AClass</class>
 <widget class="QMainWindow" name="Test10_1AClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>607</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Test10_1A</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPlainTextEdit" name="plainTextEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="separator"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionSettings"/>
   <addaction name="separator"/>
   <addaction name="actionHelp"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>607</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionSettings"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="test10_1a.qrc">
     <normaloff>:/Test10_1A/Resources/new.png</normaloff>:/Test10_1A/Resources/new.png</iconset>
   </property>
   <property name="text">
    <string>新建</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="test10_1a.qrc">
     <normaloff>:/Test10_1A/Resources/save.png</normaloff>:/Test10_1A/Resources/save.png</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="test10_1a.qrc">
     <normaloff>:/Test10_1A/Resources/setting.png</normaloff>:/Test10_1A/Resources/setting.png</iconset>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="test10_1a.qrc">
     <normaloff>:/Test10_1A/Resources/help.png</normaloff>:/Test10_1A/Resources/help.png</iconset>
   </property>
   <property name="text">
    <string>帮助</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="test10_1a.qrc"/>
 </resources>
 <connections/>
</ui>
